transcom/mymove
# B 20287 int

## [B-20287](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A972120)

## Summary

AOA packet can now be downloaded when the Advance Request is in an Edited status. I believe recent changes to the behavior of when Advance Requests are labeled as Edited vs Rejected allows this to be a simple solution.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the Milmove App
2. Login as a Customer
3. Create a new PPM
4. Login as a Service Counselor
5. Access the PPM you created
6. Approve the shipment, but on the Advance Request screen be sure to change the requested amount. This will trigger the 'EDITED' status
7. Verify the AOA packet can be downloaded for both the SC and Customer.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
